### ISSUE TITLE
Diff is not working

### ISSUE THREAD

**Issue by yurii-polishchuk** [daumiller/P4Gutter#2]
Hello,

Thanks for this project, you did a great job. I am trying to use this package, installed it through PackageControl, setup and got an error.

Maybe my perforce client is not correctly setup? or something wrong with params? 
Because Perforce Visual Client works fine.

SublimeText3 Log

```
C:\Program files\Perforce\p4.exe diff -dl C:\Users\<username>\Perforce\<somename>\depot\apps\test\source\Utils.js
P4Gutter Error: "C:\Users\<username>\Perforce\<somename>\depot\apps\test\source\Utils.js - file(s) not opened on this client.".
```

CMD

```
C:\>p4 set
P4CLIENT=<somename> (set)
P4EDITOR=C:\Windows\SysWOW64\notepad.exe (set)
P4PASSWD=<somepass> (set)
P4PORT=perforce.<somename>:1666 (set)
P4USER=<username> (set)
```

UserSetting

```
{
    // Add path to your Perforce command line binary here.
    // Usually "/usr/bin/p4" for OSX.
    "binary": "C:\\Program files\\Perforce\\p4.exe",

    // Perforce user name.
    // Should be the same as your "P4USER" environment variable, if set.
    "user": "<username>",

    // Perforce server/port.
    // Should be the same as your "P4PORT" environment variable, if set.
    "port": "perforce.<somename>:1666",

    // Perforce workspace.
    // You should probably use a ".p4_workspace" file in your workspace root directory to set this.
    // But you can set it globally here, if desired.
    "workspace": "<somename>",

    // Enabled
    // Disable the entire plugin here (until a better timeout fix is implemented).
    "enabled": true,

    // Log errors to Sublime console.
    // This may be useful for initial setup, but is really just noise after.
    "errorlog": true
}
```

**Comment by daumiller** [comment]
Thanks for the report.

It looks like your connection is fine, but there's an issue with this file or workspace.
The P4 commands don't think this file is part of your client/workspace/project. If the file isn't yet part of your project, then there's nothing to diff against (and it should be fine to ignore the error, if it's not affecting anything). But if the file is part of your project, I would guess that your workspace name isn't being set correctly.
- Have you tried running the same diff command that was logged? Does it give you a different result?
- Have you tried using a ".p4_workspace" file in the project directory (instead of specifying the workspace in your P4Gutter settings file)?

**Comment by yurii-polishchuk** [comment]
Thanks for quick answer, I really appreciate!

File is a part of my project and I am in workspace dir. Also UI client works ok with same file.(diff in visual client, etc)
- I didn't tried to run this command in cmd, sorry about that. But when I tried, there was a same error... I think something wrong with my p4 environment, I will investigate this. (it's weird to me that UI client work ok)
- yes, I also tried to create ".p4_workspace" file with workspace name in it.
